### ISSUE TITLE
Add "how to cite" section to documentation

### ISSUE THREAD

**Issue by jgosmann** [nengo/nengo#1113]
It would probably be good to have a section in the documentation that tells people how to cite Nengo if they use it their work. This section should also state that the version number should be included as newer versions are not always 100% backwards compatible.

**Comment by tbekolay** [comment]
Agreed... right now we have `CITATION.rst` in the main folder, so I would suggest that we:

- [ ] Add info about including version numbers in `CITATION.rst`
- [ ] Include `CITATION.rst` in the documentation (using the `include` directive)

**Comment by Seanny123** [comment]
What is this `include` directive you speak of?

**Comment by tbekolay** [comment]
> What is this include directive you speak of?

It's in [docutils](http://docutils.sourceforge.net/docs/ref/rst/directives.html#including-an-external-document-fragment). We use it to include the license text [here](https://github.com/nengo/nengo/blob/master/docs/license.rst).
